src/main/scala/scala/tools/refactoring/implementations/oimports/ImportsOrganizer.scala

```diff
@@ -0,0 +1,79 @@
+package scala.tools.refactoring
+package implementations.oimports
+
+import scala.tools.nsc.Global
+
+abstract class ImportsOrganizers[G <: Global, U <: TreeToolbox[G]](val treeToolbox: U) {
```

Comment:
sorry for confusion but now I realize why there is a nested class (is quite late for me now). So I just need something like:
`class ImportsOrganizer[G <: Global, U <: TreeToolbox[G], T <: Global.Tree]...`
the `T` to parameterize `forTreesOf` but maybe is better construction and Simon or Iulian will suggest it.